TrackingTools/TrackFitters/python/RungeKuttaFitters_cff.py

```diff
@@ -1,29 +1,26 @@
 import FWCore.ParameterSet.Config as cms
 
-import TrackingTools.TrackFitters.KFTrajectoryFitterESProducer_cfi
-RKTrajectoryFitter = TrackingTools.TrackFitters.KFTrajectoryFitterESProducer_cfi.KFTrajectoryFitter.clone(
-    ComponentName = cms.string('RKFitter'),
-    Propagator = cms.string('RungeKuttaTrackerPropagator')
-)
-
-import TrackingTools.TrackFitters.KFTrajectorySmootherESProducer_cfi
-RKTrajectorySmoother = TrackingTools.TrackFitters.KFTrajectorySmootherESProducer_cfi.KFTrajectorySmoother.clone(
-    ComponentName = cms.string('RKSmoother'),
-    Propagator = cms.string('RungeKuttaTrackerPropagator')
-)
-
-import TrackingTools.TrackFitters.KFFittingSmootherESProducer_cfi
-RKFittingSmoother = TrackingTools.TrackFitters.KFFittingSmootherESProducer_cfi.KFFittingSmoother.clone(
-    ComponentName = cms.string('RKFittingSmoother'),
-    Fitter = cms.string('RKFitter'),
-    Smoother = cms.string('RKSmoother')
-)
-
-KFFittingSmootherWithOutliersRejectionAndRK = RKFittingSmoother.clone(
-    ComponentName = cms.string('KFFittingSmootherWithOutliersRejectionAndRK'),
-    EstimateCut = cms.double(20.0),
-    # ggiurgiu@fnal.gov : Any value lower than -15 turns off this cut.
-    # Recommended default value: -14.0. This will reject only the worst hits with negligible loss in track efficiency.  
-    LogPixelProbabilityCut = cms.double(-14.0),                               
-    MinNumberOfHits = cms.int32(3)
-)
+import TrackingTools.TrackFitters.KFTrajectoryFitter_cfi
+RKTrajectoryFitter = TrackingTools.TrackFitters.KFTrajectoryFitter_cfi.KFTrajectoryFitter.clone()
+RKTrajectoryFitter.ComponentName = 'RKFitter'
+RKTrajectoryFitter.Propagator = 'RungeKuttaTrackerPropagator'
+
+
+import TrackingTools.TrackFitters.KFTrajectorySmoother_cfi
+RKTrajectorySmoother = TrackingTools.TrackFitters.KFTrajectorySmoother_cfi.KFTrajectorySmoother.clone()
+RKTrajectorySmoother.ComponentName = 'RKSmoother'
+RKTrajectorySmoother.Propagator = 'RungeKuttaTrackerPropagator'
+
+
+import TrackingTools.TrackFitters.KFFittingSmoother_cfi
+RKFittingSmoother = TrackingTools.TrackFitters.KFFittingSmoother_cfi.KFFittingSmoother.clone()
+RKFittingSmoother.ComponentName = 'RKFittingSmoother'
+RKFittingSmoother.Fitter = 'RKFitter'
+RKFittingSmoother.Smoother = 'RKSmoother'
+
+
+KFFittingSmootherWithOutliersRejectionAndRK = RKFittingSmoother.clone()
+KFFittingSmootherWithOutliersRejectionAndRK.ComponentName = 'KFFittingSmootherWithOutliersRejectionAndRK'
+KFFittingSmootherWithOutliersRejectionAndRK.EstimateCut = 20.0                
+KFFittingSmootherWithOutliersRejectionAndRK.MinNumberOfHits = 3
```

Comment:
LogPixelProbabilityCut now becomes 0 (if I read fillDescriptions correctly)
Wouldn't it significantly affect users of this fit/smoother?